doc/008-debugging.md

```diff
@@ -0,0 +1,48 @@
+# Debugging
+
+As you are developing your custom Hypersync app you may find that your app isn't working the way that you intended. In these situations you will need to debug your app to figure out what is going wrong.
+
+The Hypersync SDK provides a logging infrastructure that will allow you to monitor your app as it executes. This can be an effective means of debugging many problems.
+
+The Network tab in your browser's Developer Tools area can also be helpful.
+
+## Logging
+
+The SDK provides a `Logger` class that you can use to write out logging information as your app runs. The Logger class has four logging methods that differ only in the severity that is attached to the log output:
+
+```
+Logger.debug(message, detail)
+Logger.info(message, detail)
+Logger.debug(message, detail)
+Logger.debug(message, detail)
```

Comment:
Should this be the following?
```
Logger.error
Logger.warn
```

Reply:
Whoops.  Looks like I didn't finish this edit.

Reply:
Fixed.